dev/TeachingTip/InteractionTests/TeachingTipTests.cs

```diff
@@ -465,6 +465,46 @@ public void TipsWhichDoNotFitDoNotOpen()
             }
         }
 
+        [TestMethod]
+        public void VerifyTheming()
+        {
+            using (var setup = new TestSetupHelper("TeachingTip Tests"))
+            {
+                elements = new TeachingTipTestPageElements();
+                var actionButtonComboBox = elements.GetActionButtonContentComboBox();
+                actionButtonComboBox.SelectItemByName("Small text");
+                elements.GetSetActionButtonContentButton().Invoke();
+
+                ScrollTargetIntoView();
+                ScrollBy(10);
+
+                elements.GetShowButton().Invoke();
+
+                Wait.ForIdle();
+
+                Verify.AreEqual("#FF000000", elements.GetEffectiveForegroundOfTeachingTipButtonTextBlock().GetText(), "Default button foreground should be black");
+                Verify.AreEqual("#FF000000", elements.GetEffectiveForegroundOfTeachingTipContentTextBlock().GetText(), "Default content foreground should be black");
+
+                // Change to Dark, make sure the font switches to light
+                var themingComboBox = elements.GetThemingComboBox();
+                themingComboBox.SelectItemByName("Dark");
+
+                Wait.ForIdle();
```

Comment:
I'd make this a helper as well.

Reply:
Since it's just used here I don't find this hard to read, I think another function adds unnecessary indirection. I did it for the 3 lines at the top.

---
In reply to: [279937810](https://github.com/Microsoft/microsoft-ui-xaml/pull/637#discussion_r279937810) [](ancestors = 279937810)